components/DateTime.js

```diff
@@ -1,21 +1,49 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import ReactDateTime from 'react-datetime';
+import DateTimePicker from 'react-datetime-picker/dist/entry.nostyle';
+import styled from 'styled-components';
 
 import dayjs from '../lib/dayjs';
 
+const DateTimePickerWrapper = styled.div`
+  width: 100%;
+  height: 34px;
+  padding: 6px 12px;
+  background-color: #fff;
+  background-image: none;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+
+  .react-datetime-picker__wrapper {
+    border: none;
+  }
+`;
+
 class DateTime extends React.Component {
   static propTypes = {
     date: PropTypes.string.isRequired,
     timezone: PropTypes.string.isRequired,
+    timeFormat: PropTypes.bool.isRequired,
   };
 
   render() {
     const props = this.props;
-    const { date, timezone } = props;
+    const { date, timezone, timeFormat } = props;
     const value = dayjs(new Date(date)).tz(timezone);
 
-    return <ReactDateTime value={value.isValid() ? value.toDate() : ''} utc={timezone === 'utc'} {...props} />;
+    return (
+      <DateTimePickerWrapper>
+        <DateTimePicker
+          style={{ border: 'none' }}
+          format={timeFormat ? 'MM/dd/y h:mm a' : 'MM/dd/y'}
```

Comment:
We shouldn't force the format to be US, it's a confusing one for most European people who use `dd/MM/y`. `react-intl` may have some helpers to provide with that, maybe through `FormattedDate`?

Today unless I'm mistaking this input is only used for event dates (startsAt, endsAt) which are datetimes. So I think we should make this the default to keep it simple for now, if we need a date picker we can always implement it later.

Reply:
@Betree : Thanks for pointing it out. We are also using this in the Teams page (see below). The problem I am trying to solve here is to only show the calendar for the Teams page and show both calendar and the clock for the events page. I've solved this by importing both DateTimePicker and DatePicker and using the locale from `react-intl`. Take a look and let me know if there's any other issues. 

Also my [previous comment](https://github.com/opencollective/opencollective-frontend/pull/5620#issuecomment-755375428) about the Firefox problem still stands I think when we are selecting times in the events page. Let me know if you have any suggestions on this one. 🤔 

![image](https://user-images.githubusercontent.com/12435965/104041309-b4c54d80-518d-11eb-919f-019e7da178f7.png)